Source/WebCore/inspector/InspectorStyleSheet.cpp

```diff
@@ -1403,6 +1435,9 @@ RefPtr<Protocol::CSS::CSSRule> InspectorStyleSheet::buildObjectForRule(CSSStyleR
     if (groupingsPayload->length())
         result->setGroupings(WTFMove(groupingsPayload));
 
+    if (auto&& sourceData = ruleSourceDataFor(rule))
```

Comment:
does this actually return a `&&`?  a bit odd for us to explicitly say that without actually `WTFMove` or anything

Reply:
This appears to be a typo on my part. I can't figure out a reason I would have written this 😅